cli/formatter/unordered_map.go

```diff
@@ -0,0 +1,22 @@
+package formatter
+
+type unorderedMap[T comparable] struct {
```

Comment:
It would be nice to have `ForEach()` method here. It could be `orderedMap` than.

Reply:
So you suggesting something like that?
```go
func (u *unorderedMap[T]) ForEach() map[T]any {
	return u.innerMap
}
```
Or like
```go
func (u *unorderedMap[T]) ForEach(f func(key T, value any)) {
	for _, key := range u.keys {
		f(key, u.innerMap[key])
	}
}
```

Reply:
The second.

Reply:
I don't know if this is necessary since unorderedMap is also private. Or do you want to make it public?

Reply:
Ok, added this method. Also used it in some `table.go` functions.